nemo_skills/pipeline/megatron_lm/train.py

```diff
@@ -208,6 +213,17 @@ def train_megatron_lm(
         extra_arguments=extra_arguments,
     )
 
+    # Add extra sbatch arguments
+    slurm_kwargs = {"exclusive": exclusive} if exclusive else {}
+    if sbatch_arguments:
+        try:
+            sbatch_kwargs = json.loads(sbatch_arguments)
+            slurm_kwargs.update(sbatch_kwargs)
+        except json.JSONDecodeError as e:
```

Comment:
let's also check if it's already a dict, then we skip json conversion. This way people can provide it as both json-string or a dict which is more convenient when launching from python